projs/test-game/inc/GameModule.h

```diff
@@ -1,9 +1,4 @@
-//
-// Created by dpete on 02/09/2022.
-//
-
-#ifndef UMBRA_GAMEMODULE_H
-#define UMBRA_GAMEMODULE_H
+#pragma once
```

Comment:
Isn't ``pragma once`` msvc only?

Reply:
Given that I'm using minge to compile, no.